dist/images/install.sh

```diff
@@ -2010,7 +2010,6 @@ spec:
                   not:
                     enum:
                       - int
-                      - external
```

Comment:
This change should also be applied to `kubeovn-helm/templates/kube-ovn-crd.yaml`.

Reply:
ok